### PR TITLE
Add documentation to v2

### PR DESCRIPTION
**Pull request by anatbal**
This documentation PR includes:
1. Updated diagram
2. Deployment Guide which details the whole process and each parameters and it description
3. Updated Overview
4. Updated deployment button + json files
- Small refactor of  postgresOMOPCDMpassword  name - so will be presented more clear on deployment screen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,29 @@ Some of the OHDSI projects included:
 * [Achilles](https://github.com/OHDSI/Achilles) - provides descriptive statistics on an OMOP CDM database
 * [ETL-Synthea](https://github.com/OHDSI/ETL-Synthea) - Conversion from Synthea CSV to OMOP CDM
 
+## Overview
+
+![alt text](./docs/media/OHDSIonAzure.png "Architecture")
+
+Once deployed, The OHDSI on Azure solution will have the following components:
+
+1. Your Atlas and Webapi instances will be deployed as container on Azure App Service. The images get pulled from Docker Hub. Both app services will be deployed in a single App Service Plan.
```
Comment (tamirkamara) [inline README.md:35]:
```suggestion
1. An Atlas and Webapi instances will be deployed as container on Azure App Service. The images get pulled from Docker Hub. Both sites will be deployed into a single App Service Plan.
```

File: README.md [hunk 2]
```diff
@@ -26,15 +26,29 @@ Some of the OHDSI projects included:
...
+2. A Storage Account will be deployed with a filesystem. This is used to store the Atlas configuration files.
+3. Azure database for PostgreSQL will be deployed with different databases - one for the OMOP CDM and one for the Atlas WebAPI schemas.
+4. Key Vault will be deployed to store the database credentials and other secrets.
+5. Log Analytics workspace will be deployed to collect logs from the App Services, PostgreSQL and Key Vault.
```
Comment (tamirkamara) [inline README.md:39]: All items on this list are deployed as mentioned in the line before the list. I think you can remove it from all items here.

File: docs/DeploymentGuide.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| Resource Group                | The name of the resource group.                                                                               | N/A                                                           |
+| Region                        | The region of the resource group.                                                                             | As the resource group                                         |
+| Location                      | The location of the resources.                                                                                | Resource group location                                       |
+| Suffix                        | The suffix for the resource names.                                                                            | Unique string is being generated                              |
+| CDM Container URL             | The URL of the CDM container. This could be replaced with any container containing any CDM tables.           | <https://omoppublic.blob.core.windows.net/shared/synthea1k/> |
```
Comment (tamirkamara) [inline docs/DeploymentGuide.md:25]:
```suggestion
| CDM Container URL             | The URL prefix or Blob container where CDM files can be downloaded from.           | <https://omoppublic.blob.core.windows.net/shared/synthea1k/> |
```

File: docs/DeploymentGuide.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+| CDM SAS Token                 | The SAS token for accessing the CDM container. If no need for a SAS token, leave empty.                      | None                                                          |
+| Postgres OMOP CDM Database Name| The name of the PostgreSQL OMOP CDM database.                                                                 | None                                                          |
```
Comment (tamirkamara) [inline docs/DeploymentGuide.md:27]: Should we add a default here?

File: docs/DeploymentGuide.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+| Postgres OMOP CDM Password     | The password for the PostgreSQL OMOP CDM user.                                                                | Unique password is being generated                            |
+| Atlas Security Admin Password | The password for the Atlas security admin user.                                                                | Unique password is being generated                            |
+| Atlas Users List              | The list of users for the Atlas system should be provided in the following format: 'username1,password1,username2,password2,' and so on. Separate each set of credentials with a comma. For example, if you have two sets of credentials, it should look like this: 'user1,pass1,user2,pass2.' Ensure that the usernames and passwords are in the correct order and do not include any additional spaces or characters between the credentials. Note that this format allows you to specify multiple sets of username-password pairs for different users. Make sure each pair is properly formatted and separated by commas. | None                                                          |
```
Comment (tamirkamara) [inline docs/DeploymentGuide.md:36]: Will the first user be an Admin? Should we say which roles will those get?

```suggestion
| Atlas Users List              | The list of users for the Atlas system should be provided in the following format: 'username1,password1,username2,password2' and so on. Ensure usernames and passwords are in the correct order and do not include any additional spaces or characters between the credentials. Note that this format allows you to specify multiple sets of username-password pairs for different users. Make sure each pair is properly formatted and separated by commas. | None                                                          |
```

**Review by tamirkamara — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (anatbal) [reply README.md]: Deleted

File: README.md [hunk 6]
```diff
@@ -26,15 +26,29 @@ Some of the OHDSI projects included:
...
+
+* You can host your CDM in Azure PostgreSQL. You can load your vocabularies into Azure Storage Container as cs.gz files, and pass as a paramater in your custom deployment.
```
[the remark this answers is not in the document]
Comment (anatbal) [reply README.md]: Fixed

File: docs/DeploymentGuide.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
+# Deployment Guide
+
+## Prerequisites
+
+In order to deploy OHDSI on Azure, you will need the following prerequisites:
+
+1. Azure Subscription
+2. Resource Group
```
[the remark this answers is not in the document]
Comment (anatbal) [reply docs/DeploymentGuide.md]: Deleted

File: docs/DeploymentGuide.md (hunk 4 above)
Comment (anatbal) [reply docs/DeploymentGuide.md]: Added default for both SAS and DB name

File: docs/DeploymentGuide.md (hunk 5 above)
Comment (anatbal) [reply docs/DeploymentGuide.md]: Fixed

File: docs/DeploymentGuide.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+| Atlas Security Admin Password | The password for the Atlas security admin user.                                                                | Unique password is being generated                            |
+| Atlas Users List              | The list of users for the Atlas system should be provided in the following format: 'username1,password1,username2,password2,' and so on. Separate each set of credentials with a comma. For example, if you have two sets of credentials, it should look like this: 'user1,pass1,user2,pass2.' Ensure that the usernames and passwords are in the correct order and do not include any additional spaces or characters between the credentials. Note that this format allows you to specify multiple sets of username-password pairs for different users. Make sure each pair is properly formatted and separated by commas. | None                                                          |
+| Local Debug                   | Enable local debugging mode.                                                                                   | false                                                         |
```
[the remark this answers is not in the document]
Comment (anatbal) [reply docs/DeploymentGuide.md]: Added

**Review by tamirkamara — APPROVED** [review]

**Pull request merged**